src/components/Avatar/Avatar.tsx

```diff
@@ -70,55 +72,43 @@ const AvatarStatusItems: FC<BaseAvatarProps> = React.forwardRef(
     const htmlDir: string = useCanvasDirection();
 
     const getStatusItemPositionStyle = (
-      itemPos: StatusItemsPosition,
-      itemProps: StatusItemsProps,
-      wrapperPadding: string | number
+      itemPos: StatusItemsPosition
     ): React.CSSProperties => {
       const outlineWidth: string = outline?.outlineWidth ?? '0px'; // Avatar outline width
-      const avatarWidth: string = size;
-      const itemWidth: string = `(${itemProps.size} + (2 * ${wrapperPadding}))`; // Status item width
+      const outlineOffset: string = outline?.outlineOffset ?? '0px'; // Avatar outline offset
+      const radius: string = `calc((${size} + ${outlineWidth} + ${outlineOffset}) / 2)`;
```

Comment:
`AvatarGroup` items require outlines too, in your screen shot the outline was removed - will this change also work when group items have outlines to indicate status?

Reply:
See comments below.

Reply:
Yes, it will work